danielabar/meblog
# feature/learning page

- Introduce new learning page, directories for content, and remark plugin config to locate new content
- add some more learning content and update test snapshots for new learning menu item
- more course data and verify images can be displayed
- add course list and course components, start laying out course card info
- switch to Figtree font for course cards, style category tag and course title
- upgrade react icons and style remaining course card content
- update todo list for learning feature
- brainstorm on some content for learning intro paragraph
- update test snapshots as upgrading react-icons changed svg paths, even though icons still look the same
- add learning intro component for top of learning page
- start on category tag colors
- experiment with some imagemagick commands
- update card shadows
- add remainder of pluralsight courses, wesbos, mongou, attempt at category color scheme
- import course data from tutsplus
- implement color scheme for course category tags
- small updates to learning intro wording
- wip on adding course images for each card
- adjust course image brightness and on hover
- add more images for course cards
- populate remaining course card images
- seo for learning page and remove console log for each course
- reference test page
- fix class should be className
- add test for learning page
- test learn intro component
- test course card component
- update course card test to verify notes link
- test course list component
- add kafka course to learning cards
- use different image for learning page seo
- small tweak to meta description for learning page
- remove not doing tasks
